### ISSUE TITLE
ida -> idas, cvode -> cvodes

### ISSUE THREAD

**Issue by mottelet** [LLNL/sundials#333]
Hello,

Scilab uses cvode and ida as solvers in Xcos (block diagram simulator) and we plan to make available sensitivity computation in the future. Since cvode and cvodes are mutually incompatible (ida and idas either) but cvodes and idas can be used seemlessly without sensitivity computation, we are wondering if there is any risk of regression if we just change our cvode and ida headers includes for cvodes and idas ones (and of course link against cvodes and idas libraries). What do you think about it ?

Thanks for your help.

Stéphane, on behalf of the Scilab Team

**Comment by balos1** [comment]
I would not expect there to be any regressions, but if you come across some please report them. We try to keep the sensitivity variants completely up to date with the non-sensitivity variants in terms of features and bug fixes.

**Comment by mottelet** [comment]
Thank you.

**Comment by mottelet** [comment]
We have a little problem. We use cvode and ida from the same function, but when we replace respective includes by cvodes and idas ones there is a macro redefinition warning and a typedef redefinition error we did not have before:
```
../../modules/differential_equations/src/patched_sundials/include/../src/idas/idas_impl.h:61:9: warning: 'ETA_MAX_FX_DEFAULT' macro redefined [-Wmacro-redefined]
../../modules/differential_equations/src/patched_sundials/include/../src/idas/idas_impl.h:64:9: warning: 'ETA_MIN_DEFAULT' macro redefined [-Wmacro-redefined]
../../modules/differential_equations/src/patched_sundials/include/../src/idas/idas_impl.h:589:8: error: redefinition of 'CkpntMemRec'
../../modules/differential_equations/src/patched_sundials/include/../src/idas/idas_impl.h:662:8: error: redefinition of 'DtpntMemRec'
../../modules/differential_equations/src/patched_sundials/include/../src/idas/idas_impl.h:668:16: error: redefinition of 'HermiteDataMemRec'
../../modules/differential_equations/src/patched_sundials/include/../src/idas/idas_impl.h:673:4: error: typedef redefinition with different types ('struct (unnamed struct at ../../modules/differential_equations/src/patched_sundials/include/../src/idas/idas_impl.h:668:16) *' vs 'struct HermiteDataMemRec *')
../../modules/differential_equations/src/patched_sundials/include/../src/idas/idas_impl.h:676:16: error: redefinition of 'PolynomialDataMemRec'
../../modules/differential_equations/src/patched_sundials/include/../src/idas/idas_impl.h:685:4: error: typedef redefinition with different types ('struct (unnamed struct at ../../modules/differential_equations/src/patched_sundials/include/../src/idas/idas_impl.h:676:16) *' vs 'struct PolynomialDataMemRec *')
```

**Comment by mottelet** [comment]
The typedefs should be prefixed by "CVode" and "IDA" (that's what I will do as a rapid fix)

**Comment by balos1** [comment]
Fixed by #337.
